desktop/updater/manual-updater/index.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * External Dependencies
+ */
+const electron = require( 'electron' );
+const https = require( 'https' );
+
+const dialog = electron.dialog;
+const shell = electron.shell;
+
+function ManualUpdater( url ) {
+	this.hasPrompted = false;
+	this.url = url;
+}
+
+ManualUpdater.prototype.ping = function() {
+	const that = this;
+
+	https.get( this.url, function( response ) {
+		let body = '';
+
+		response.on( 'data', function( chunk ) {
+			body += chunk;
+		} );
+
+		response.on( 'end', function() {
+			if ( body ) {
+				let update = JSON.parse( body );
```

Comment:
this could throw an exception, which is probably something we want to guard against with a `try` block so that a network or server issue doesn't kill the app